x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -88,15 +87,15 @@ export const SearchExperience: React.FC = () => {
   const connector = new AppSearchAPIConnector({
     cacheResponses: false,
     endpointBase,
-    engineName: engine.name,
+    engineName: engine.name as string,
     additionalHeaders: {
       'kbn-xsrf': true,
     },
-  });
+  } as ConstructorParameters<typeof AppSearchAPIConnector>[0]);
```

Comment:
This is a hack that silences TS error from unknown `cacheResponses` field. This is a bug in Search UI types, added to issue: https://github.com/elastic/enterprise-search-team/issues/2226